### PR TITLE
Add single source of truth Cosmos and Blockchain glossary to cosmos.network website

### PR DESCRIPTION
**Pull request by lovincyrus**
The ultimate goal is to have a single source of truth for the glossary.

This JSON file aims to be the one and only glossary list used across our front-facing websites (cosmos, tendermint, docs, etc.). We're currently still relying on the `v1.cosmos.network`'s /intro and /gossary, and in the interim, we'll use this `dict.json` to display the definition tooltip on the new [`cosmos/cosmos.network`](https://github.com/cosmos/cosmos.network).

Once we build the new `/intro` and `/glossary` on the new cosmos.network, we'll deprecate the [Airtable Glossary](https://airtable.com/tbltEIRH9YJ8yE3PP/viw5ohOPNAk6ld1Ig?blocks=hide) and [cosmos-ui's JSON list](https://github.com/tendermint/cosmos-ui/blob/develop/src/Tooltip/dict.json).

---

Ref: https://github.com/cosmos/advocates/issues/13#issuecomment-817016663

* create glossary folder
* add dict.json
* add readme

### CODE REVIEW

File: glossary/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Glossary
```
Comment (barriebyron) [inline glossary/README.md]:
```suggestion
# Cosmos Blockchain Glossary
```

Comment (barriebyron) [reply glossary/README.md]: to improve our SEO, let's include Cosmos and Blockchain in the title of the glossary. (Is there a way to improve our findability? more metadata perhaps?)

**Review by barriebyron — DISMISSED** [review]
thank you @lovincyrus for this work! We can add terms and curate our editorial standards according to the [Glossary Guidelines](https://www.notion.so/allinbits/Glossary-Guidelines-ab9e20306b0f4aae953f2acdc259aea8) that we work toward

File: glossary/README.md (hunk 1 above)
Comment (lovincyrus) [reply glossary/README.md]: Hmm - to improve our discoverability, we'll need to tweak the meta title and description of `cosmos.network/glossary`. Once that's done, we'll add `cosmos.network/glossary` to [the sitemap](http://cosmos.network/sitemap.xml). So Google can tell where to crawl the URLs for the specific site.

**Review by barriebyron — APPROVED** [review]
at what stage do we tweak the metadata and title and other whatsits to improve our SEO? next round? lmk if or how I can help, thank you for this work

File: glossary/README.md (hunk 1 above)
Comment (barriebyron) [reply glossary/README.md]: meta title and description tweaking, I'm up for that! @lovincyrus how can we tweak?

**Comment by lovincyrus** [comment]
> at what stage do we tweak the metadata and title and other whatsits to improve our SEO? next round?

I'll ping you when we need help with the metadata for the new Intro and Glossary page. 👌🏼

**Comment by lovincyrus** [comment]
Need +1 approving review to merge this.

**Comment by nylira** [comment]
I can't review, but +1 from me

**Review by barriebyron — APPROVED** [review]
thank you @lovincyrus

File: glossary/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Cosmos Blockchain Glossary
```
Comment (barriebyron) [inline glossary/README.md:1]:
```suggestion
<!--
  Title: Cosmos Blockchain Glossary
  Description: A curated list of blockchain terms, words, and language that are used in the Cosmos universe. 
  -->
# Cosmos Blockchain Glossary
```

Comment (barriebyron) [reply glossary/README.md:1]: is this metadata useful at all for SEO? if not, ignore. 
In regular markdown, we use metadata in the docs to help SEO

**Comment by barriebyron** [comment]
@lovincyrus do we need another issue for the navigation aids? would love to have a letter navigation aid?
A B C 
and so on 
so the user can easily navigate to the word they are looking for

File: glossary/README.md (hunk 2 above)
Comment (lovincyrus) [reply glossary/README.md:1]: We shouldn't include any YAML front matter in this readme file.

**Comment by barriebyron** [comment]
@toschdev can you add your approval so we can merge?

**Review by toschdev — APPROVED** [review]

**Comment by barriebyron** [comment]
thanks @lovincyrus and @toschdev !

**Pull request merged**